data/changelog/ocaml-lsp/2024-07-31-ocaml-lsp-1.19.0.md

```diff
@@ -0,0 +1,17 @@
+---
+title: OCaml-LSP 1.19.0
+tags: [ocaml-lsp, platform]
+changelog: |
+## Features
+
+- Add custom [`ocamllsp/getDocumentation`](/ocaml-lsp-server/docs/ocamllsp/getDocumentation-spec.md) request (ocaml/ocaml-lsp#1336)
+- Add support for OCaml 5.2 (ocaml/ocaml-lsp#1233)
+
+## Fixes
+
+- Kill unnecessary OCamlFormat processes with `sigterm` rather than `sigint` or `sigkill` (ocaml/ocaml-lsp#1343)
+---
+
+We are pleased to announce the release of OCaml LSP 1.19.0. This version brings
+official support for OCaml 5.2. It also includes a new custom query to get

```

Comment:
```suggestion
    ## Features
    
    - Add custom [`ocamllsp/getDocumentation`](/ocaml-lsp-server/docs/ocamllsp/getDocumentation-spec.md) request (ocaml/ocaml-lsp#1336)
    - Add support for OCaml 5.2 (ocaml/ocaml-lsp#1233)
    
    ## Fixes
    
    - Kill unnecessary OCamlFormat processes with `sigterm` rather than `sigint` or `sigkill` (ocaml/ocaml-lsp#1343)
    ---
    
    We are pleased to announce the release of OCaml LSP 1.19.0. This version brings
    official support for OCaml 5.2. It also includes a new custom query to get

```